src/TouristInfoCards.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+function TouristInfoCards(props) {
```

Comment:
You left 3 divs that belong to `TouristInfoCards` in the `App.js`, I think they should be inside this component and then you could map over `cities` inside those divs. I also think that lines 6 to 16 could be extracted to a new `Card` component that receives city info as props.

Well done with extracting cities data to json file, you could import it directly in this component - there is no need to pass it from the App as props.

Reply:
and done.